modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationEndpoints/CreateAnAPIApplicationEndpoint.testcase

```diff
@@ -58,16 +58,50 @@ definition {
 		}
 	}
 
-	@priority = 5
-	test CanCreateEndpointWithSiteScope {
+	@priority = 4
+	test CanCreateEndpointWithPathAndDescriptionWithSuccessAlert {
 		property portal.acceptance = "true";
 
-		task ("Given the API Application is created") {
-			ApplicationAPI.createAPIApplication(
-				baseURL = "test2",
+		task ("And Given I go to 'Endpoints' tab in 'Edit API Application'") {
+			APIBuilderUI.switchToRelatedEntryInEditApplication(
+				entity = "Endpoints",
 				title = "test2");
 		}
 
+		task ("And Given I click 'Add API Endpoint' button") {
+			Click(locator1 = "Button#PLUS");
+		}
+
+		task ("When I enter a path and Scope to create the API endpoint") {
+			Click(
+				key_value = "Select Scope",
+				locator1 = "APIBuilder#DROPDOWN_TOGGLE");
```

Comment:
Could you try to use our existing macro to replace using 'Click' and 'Type'? if we don't have you can create a new one. Thanks!

Reply:
Done, I needed to create a new macro. Thanks for review @Hijky